onyo/argparse_helpers.py

```diff
@@ -1,15 +1,29 @@
+from __future__ import annotations
+
 import argparse
 import os
+from typing import TYPE_CHECKING
 
-from typing import Sequence
+if TYPE_CHECKING:
+    from typing import Sequence
 
 
 class StoreKeyValuePairs(argparse.Action):
+    r"""Store a list of dictionaries."""
```

Comment:
Looking at the example of https://numpydoc.readthedocs.io/en/latest/format.html#short-summary it looks like even one-liner short-summary docstrings have the `"""` in the next line.

Reply:
Thanks for the catch. I have committed the fix and pushed.